Yoast/wordpress-seo
# Refactor how we handle OpenGraph images

## Summary

This PR can be summarized in the following changelog entry:

* Output width and height in OpenGraph meta tags way more often.
* When an image found for OpenGraph is larger than 2MB, choose a smaller version.
* Cache images found in a post so we don't need to do this on every page load.
* When an image has an alt tag, output it as `og:image:alt`.

## Relevant technical choices:

* Factored out two new classes, `WPSEO_Image_Utils` and `WPSEO_Content_Images`. The former to find attachment ID's from image URLs, find correct image paths and find image file sizes. The latter to find and cache images in post content.

## Test instructions

This PR can be tested by following these steps:

* Play with images in post content, as featured image, specifically set as Facebook image and make sure it all still works as expected, and has width and height output everywhere.

## Quality assurance


Fixes #9409
Fixes #9370 
Fixes #8054 
Fixes #4400 
Fixes #5541 
Fixes #8342
